### ISSUE TITLE
Custom template modules

### ISSUE THREAD

**Issue by stefanwalther** [verbose/verb#60]
Do you have an example for `verb --set includes="my-include-templates"` ...?

Tried it both, with a local module and a path following the example of readme-includes, but didn't succeed, always get:

```async-helper-base: {%= include("extension-install") %} error: %j TypeError: Cannot read property 'render' of null```

But settings in the app seems to be correct:

```js
 store:
   { name: 'verb',
     path: 'C:\\Users\\me\\data-store\\verb.json',
     data: { includes: 'me-readme-includes' } } }
```

**Comment by stefanwalther** [comment]
Hi,

I assume the right place to fix this would be `.\lib\transforms\init\load.js', right?
I don't see any code there which would load the setting from the data-store ...

Am I missing something?

Regards
Stefan

**Comment by tunnckoCore** [comment]
@stefanwalther it would it with `this.get`, imo.

**Comment by stefanwalther** [comment]
@tunnckoCore, thx, will give it a try ...

**Comment by jonschlinkert** [comment]
sorry for taking so long on this. @doowb and I have virtually refactored the entire ecosystem underlying verb and other related projects. I'm just migrating now, so any day...

**Comment by stefanwalther** [comment]
@jonschlinkert Any news on this?

**Comment by stefanwalther** [comment]
Ping ;-)

**Comment by stefanwalther** [comment]
Hi, getting this to work would be soooooooo great ;-)
@jonschlinkert & @doowb , any ideas?

**Comment by tunnckoCore** [comment]
@stefanwalther soon :) If you not noticed they work on underlying tons of packages (see `dev` branch).
You should definitely try the `dev` branch + `verb-readme-generator`.

I opened #87 and https://github.com/verbose/verb-readme-generator/issues/2, currently there are resolving problems.

**Comment by jonschlinkert** [comment]
yes we're very close. fwiw everything is working perfectly locally for me, but we need to add support for npm custom prefixes before we release - I think that's what is breaking @tunnckoCore's build, and probably will cause issues for others.

but I have to say, @doowb and I are really amazed at how awesome this is, and how well everything is coming together as we had planned. We're very close to getting everything "done"! thx for hanging in there and staying interested!

**Comment by tunnckoCore** [comment]
> I think that's what is breaking

hm, that's remind me that i didn't try after I fixed the prefix problem. :D I'll try now.

**Comment by jonschlinkert** [comment]
> after I fixed the prefix problem.

what did you do to solve it btw?

**Comment by jonschlinkert** [comment]
the solutions you pointed to seem like overkill for this, but maybe I'm missing something

**Comment by tunnckoCore** [comment]
> what did you do to solve it btw?

just moved node executable to `/usr/local/bin`, lol

> seem like overkill for this

why? ;d it didn't break anything, which was surprising for me too.

**Comment by tunnckoCore** [comment]
Btw, problem occurred with trying latest `dev`. Errors comes from templates, I'm checking and trying to realize it, then I'll open issue there.

**Comment by jonschlinkert** [comment]
@stefanwalther have you tried using 0.9? this was implemented

**Comment by stefanwalther** [comment]
Hi @jonschlinkert, what is 0.9, the dev branch? I'll give it a try immediately, really awaiting that feature ;-)

**Comment by stefanwalther** [comment]
Installing 0.9 (the dev branch), gives me the following errors:

```js
npm install verbose/verb#dev
```

```js
verb
```

```
{ [Error: base-runner#runner Cannot find module 'minimist'] code: 'MODULE_NOT_FOUND' }
```

After installing minimist, I get some other missing dependencies ... (omit-empty, etc.)

**Comment by jonschlinkert** [comment]
@doowb and I aren't able to reproduce. Since you installed locally, it's possible that verb is trying to use outdated modules (since npm doesn't force everything to reinstall the way it used to). 

did you uninstall everything (delete `node_modules` and do cache clean) and reinstall? globally and locally?
